frontend/src/components/footer/footer.css

```diff
@@ -0,0 +1,53 @@
+.footer{
+   width: 100%; 
+   height: 356px;
+    background-color:  #252525;
+}
+
+.links{
+    margin-left: 160px;
+    float:left;
+}
+
+h1{
+    font-family: 'Lato,',sans-serif;
+font-style: normal;
+font-weight: 700;
+font-size: 16px;
+line-height: 24px;
+color: #FFFFFF;
+opacity: 0.5;
+padding-top: 31px;
+padding-bottom: 24px;
+
+}
+
+.links a{
+    font-family: 'Lato',sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 24px;
+    color: #D8D8D8;
+    padding-bottom: 12px;
+}
+
+.contact{
+    float: right;
+    margin-right: 160px;
+}
+.contact img {
+    margin-right:16.4px;
+}
+
+.contact p{
+    font-family: 'Lato',sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 24px;
+    color: #D8D8D8;
+    margin-top: 0;
+    margin-bottom: 12px;
+
+}
```

Comment:
This sign shows us that we have to add new line to a file. Please fix it anywhere you see it.